packages/core/src/application.ts

```diff
@@ -135,6 +135,9 @@ export class Application extends Context {
     request: ServerRequest,
     response: ServerResponse,
   ) {
+    // allow CORS support for all endpoints so that users
+    // can test with online SwaggerUI instance
+    response.setHeader('Access-Control-Allow-Origin', '*');
```

Comment:
Sorry if I missed it earlier, but is there a reason why we need the header to be `*`?